turbopack/crates/turbopack-ecmascript/src/analyzer/graph.rs

```diff
@@ -812,15 +802,56 @@ mod analyzer_state {
     /// intentionally private to the rest of the `Analyzer` implementation.
     pub struct AnalyzerState {
         pat_value: Option<JsValue>,
+        // A unique identifier for the current function, based on the span of the function
+        // declaration.  [None] if we are in the root scope.
+        cur_fn_ident: Option<u32>,
```

Comment:
This should be `id` and not `ident`? `ident` makes me think it's referring to an identifier node?

These ids are coming from the node span, but maybe could be sequentially assigned? I'm not sure what they're used for.

Reply:
Renamed.  This is used to construct 'unique names' for `JsValue::Argument` during linking this makes it easier to associate values passed to Calls with arguments inside the function.